buildtest/cli/inspect.py

```diff
@@ -88,65 +94,118 @@ def fetch_test_names(report, names):
     return query_builders
 
 
-def inspect_list(report, terse=None, header=None, builder=None, color=None):
+def print_builders(report):
+    """This method prints all the builders.
+
+    Args:
+        report (str): Path to report file
+    """
+
+    builders = report.builder_names()
+    for name in builders:
+        console.print(name)
+
+
+def print_terse(table, no_header=None, consoleColor=None):
+    """This method prints the buildtest inspect list in terse mode which is run via command ``buildtest inspect list --terse``
+
+    Args:
+        table (dict): Table with columns required for the ``buildtest inspect list`` command.
+        no_header (bool, optional): Determine whether to print header in terse format.
+        consoleColor (bool, optional): Select desired color when displaying results
+    """
+
+    row_entry = [table[key] for key in table.keys()]
+    transpose_list = [list(i) for i in zip(*row_entry)]
+    header = "|".join(table.keys())
```

Comment:
we could probably remove this variable declaration `header` since its used once and just do 
`console.print('|'.join(table.keys()), style=consoleColor)`